app/cdap/text/text-en.yaml

```diff
@@ -3479,3 +3479,10 @@ features:
     noMatchColumnDatatype: Selected directive supported datatype does not match which the column's datatype
     no: No
     zero: '0'
+  WranglerNewSecurityOptions:
```

Comment:
The order should be
    \t1 WranglerNewUI
        \t2 Transformations
            \t3 securityOptions
                \t4 encodeDecodeOptions
                    \t5 encode: Encode
                    \t5 decode: Decode